src/main/java/com/spotinst/sdkjava/model/requests/azure/statefulNode/StatefulNodeStateRequest.java

```diff
@@ -0,0 +1,43 @@
+package com.spotinst.sdkjava.model.requests.azure.statefulNode;
+
+public class StatefulNodeStateRequest {
+
+    private String state;
```

Comment:
State should be Enum. either pause|resume|recycle

Reply:
Done